types/ember-cli-fastboot/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for ember-cli-fastboot 2.2
+// Project: https://github.com/ember-fastboot/ember-cli-fastboot
+// Definitions by: Henry Majoros <https://github.com/hmajoros>
+//                 Dan Freeman <https://github.com/dfreeman>
+//                 Chris Krycho <https://github.com/chriskrycho>
+//                 James C. Davis <https://github.com/jamescdavis>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+import './services/fastboot';
```

Comment:
it was explained, there is a service and `fastboot` module, 
but, then this should be rather:
```suggestion
import './addon/services/fastboot';
```

Reply:
No, the definition as given here is correct. Ember's classic resolution is ESM spec compliant *but did not use Node resolution*. (Respectfully, please understand that we've been maintaining the Ember types for the last five years and know what we're doing here. 😄)

Reply:
cool, I'm not trying to derail anything, sorry for the hassle sometimes :)

Reply:
No worries, just figured I’d head off further needless back and forth on these specific lines so we can focus on the other, more productive stuff. 😅